src/components/calculator/Calculator.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+
+import Button from '../common/Button';
+
+import useExpression from '../../hooks/useExpression';
+
+import { ERROR_TEXT } from '../../constants/error';
+
+const Calculator = () => {
+  const [totalText, setTotalText] = useState(0);
+  const [writeExpression, calculateExpression, ResetExpression] = useExpression();
```

Comment:
```suggestion
  const [writeExpression, calculateExpression, resetExpression] = useExpression();
```

ResetExpression만 파스칼 케이스인데 특별한 이유가 있으실까요..?

Reply:
특별한 이유는 없고 급하게 작업하면서 실수가 있었던 것 같습니다..😭